README.rst

```diff
@@ -76,6 +72,37 @@ Feature extraction:
         <user_is_anon>: False
         <user_is_bot>: False
 
+Installation
+================
+In order to use this, you need to install a few packages first:
+
+``pip install revscoring``
+
+You'll need to download NLTK data in order to make use of language features.
+
+.. code-block:: python
+
+    >>> python
+    >>> import nltk
+    >>> nltk.download()
+    >>> Downloader> d
+    >>> Identifier> wordnet
+    >>> Downloader> d
+    >>> Identifier> omw
+    >>> Downloader> q
+    >>> exit()
+
+
+You might need to install some other dependencies depending on your operating
+system.  These are for ``scipy`` and ``numpy``.
```

Comment:
Oh!  Yeah, it looks like "python-sklearn" came from one of *your* pull requests.  :P  I'm happy to remove it.